test/parallel/test-trace-events-dns.js

```diff
@@ -0,0 +1,86 @@
+'use strict';
```

Comment:
This test should probably go into test/internet?

Reply:
Honestly I are not sure the technical differences between test/internet and test/parallel. I saw dns test code in both folders. For now, I put this under test/parallel, along with other test-trace-events-xxx test files. I don't mind to move if it should go to test/internet by definition.